templates/robotics/index.html

```diff
@@ -133,32 +176,56 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
         </div>
       </div>
     </div>
-    <p class="u-align--center"><a href="https://partners.ubuntu.com/find-a-partner?technology=robotics">See all of our robotics partners</a></p>
+    <p class="u-align--center"><a href="https://canonical.com/partners/find-a-partner?filters=robotics">See all of our robotics partners&nbsp;&rsaquo;</a></p>
   </div>
 </section>
 
 <section class="p-strip">
   <div class="u-fixed-width">
-    <p class="p-heading--2">Enhanced security for mission-critical robots</p>
-    <p>Several layers of security are stacked in Ubuntu Core to keep potential surfaces of attack as low as possible for your product deployed in the field.</p>
-    <a href="/engage/securing-ros-on-robotics-platforms-whitepaper">Read the Securing ROS on robotics platforms whitepaper&nbsp;&rsaquo;</a>
+    <h2 class="p-heading--2">Bringing Ubuntu's worldwide infrastructure to your robot</h2>
+    <h4>Deploying a robotics solution? For more than a decade, Ubuntu has provided the best development environment for robotics developers.</h4>
+    <p>Together with <a href="/core">Ubuntu Core</a> — OS optimised for decentralised computing environments — and <a href="https://snapcraft.io/docs/robotics">snap</a> — fully confined embedded containers — developers can easily build and securely deploy their devices. And for the software management of your device, the <a href="/internet-of-things/appstore">IoT Snap Store</a> allows developers to unlock the reliable and widely used update infrastructure of Ubuntu applications in their robots.</p>
+    <div class="u-fixed-width">
```

Comment:
This image looks a bit horizontally pressed for me but please disregard it if it looks okay for you! 
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/57550290/169016836-761c39b1-e500-4990-816c-879b406060c3.png">

Reply:
In your screenshot it does look a bit weird, but I don't know how to reproduce that. Maybe, can you hard reload or open the page in incognito? This is what I see:

![robot-diagram](https://user-images.githubusercontent.com/14939793/169040063-b0b9a3eb-d13b-4f93-9653-6a96d9ea1f3c.png)